lib/spack/spack/fetch_strategy.py

```diff
@@ -894,7 +894,7 @@ def fetch(self):
         args = []
         if self.revision:
             args.append('at revision %s' % self.revision)
-        tty.msg("Trying to clone Mercurial repository:", self.url, *args)
+        tty.msg("Cloning Mercurial repository:", self.url, *args)
```

Comment:
We use "Mercurial" here instead of "mercurial" or "hg", but elsewhere we use "svn" instead of "Subversion" or "subversion".

I'm in favor of using "mercurial" and "subversion".

Reply:
That makes sense. I changed them in the way you suggested.